vignettes/epipredict.Rmd

```diff
@@ -1,8 +1,8 @@
 ---
-title: "Get started with `epipredict`"
+title: "Get started with `{epipredict}`"
```

Comment:
So this breaks the formatting in the title, and makes it literally those characters.

Reply:
Ah, thanks for pointing out.